test/browser_tests/page_objects/page_careers-working-at-cfpb.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+var _assign = require( 'lodash' ).assign;
+var _getQAelement = require( '../util/QAelement' ).get;
+var careersSocialSection = require( '../shared_objects/careersSocialSection' );
+var relatedLinksSection = require( '../shared_objects/relatedLinksSection' );
+
+
+function WorkingAtCFPB() {
+
+  _assign( this, careersSocialSection, relatedLinksSection );
+
+  this.get = function() {
+    browser.get( '/careers/working-at-cfpb/' );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
+
+  this.sideNav = element( by.css( '.nav-secondary' ) );
+
+  this.introSection = _getQAelement( 'intro-section' );
+
+  this.introSectionTitle = this.introSection.element( by.css( 'h1' ) );
+
+  this.introSectionLink = this.introSection.element( by.css( 'a' ) );
+
+  this.mobileCarouselItems =
+  element.all( by.css( '.media-stack .media-stack_item' ) );
```

Comment:
`media-stack` is the BEM parent, so you probably don't need the nesting here and can just target `by.css( '.media-stack_item' )`